### PR TITLE
Make evaluate changes print why a test failed

### PR DESCRIPTION
**Pull request by skejserjensen**
This PR adds additional print statements to [Evaluate-ModelarDB-Changes/main.py](https://github.com/ModelarData/Utilities/blob/dev/evaluate-changes-print-why/Evaluate-ModelarDB-Changes/main.py) to make it simpler to understand why an experiment did not complete.

### CODE REVIEW

File: Evaluate-ModelarDB-Changes/main.py [hunk 1]
```diff
@@ -110,6 +97,16 @@ def start_modelardbd(modelardb_cargo_toml, data_folder):
     return process
 
 
+def errors_occured(output_stream):
```
Comment (CGodiksen) [inline Evaluate-ModelarDB-Changes/main.py:100]: Minor but I think it should be `occurred` instead.

File: Evaluate-ModelarDB-Changes/main.py [hunk 2]
```diff
@@ -160,24 +152,38 @@ def send_sigint_to_process(process):
...
     output_file.write('  "')
-    output_file.write(" ".join(changes))
+    output_file.write(str(current_change))
     output_file.write('": ')
     output_file.write(json.dumps(results))
     output_file.write(",\n")
     output_file.flush()
 
 
+def print_seperator(current_change, last_change):
```
Comment (CGodiksen) [inline Evaluate-ModelarDB-Changes/main.py:182]: Minor but I think it should be `separator` instead.

File: Evaluate-ModelarDB-Changes/main.py [hunk 3]
```diff
@@ -231,10 +240,11 @@ def finish_output_file_and_kill_process(output_file):
     signal.signal(signal.SIGINT, lambda _signal_number, _frame: sys.exit(0))
 
     # Evaluate changes.
-    changes_len = len(changes)
+    last_change = len(changes)
     for index, changes in enumerate(changes):
         # Print what changes are being evaluating.
```
Comment (CGodiksen) [inline Evaluate-ModelarDB-Changes/main.py:245]: Minor but maybe "Print what changes are being evaluated." instead.

**Review by CGodiksen — CHANGES_REQUESTED** [review]

File: Evaluate-ModelarDB-Changes/main.py (hunk 1 above)
Comment (skejserjensen) [reply Evaluate-ModelarDB-Changes/main.py:100]: Fixed.

File: Evaluate-ModelarDB-Changes/main.py (hunk 2 above)
Comment (skejserjensen) [reply Evaluate-ModelarDB-Changes/main.py:182]: Fixed.

File: Evaluate-ModelarDB-Changes/main.py (hunk 3 above)
Comment (skejserjensen) [reply Evaluate-ModelarDB-Changes/main.py:245]: Fixed.

**Review by CGodiksen — APPROVED** [review]

**Pull request merged**